src/components/ExpandedNode/ExpandedNode.tsx

```diff
@@ -348,6 +348,9 @@ const ExpandedNodeMemo: FC<ExpandedNodeMemoProps> = memo(
           <tbody>
             <StatusField documentation={documentation} />
             {guidance || branch}
+            {!isActionable && comments !== '' && comments != null && (
```

Comment:
Since `''` and `null` are falsy you can condense the last two.
```suggestion
            {!isActionable && comments && (
```